config/opensearch_dashboards.yml

```diff
@@ -17,7 +17,8 @@
 #server.rewriteBasePath: false
 
 # The maximum payload size in bytes for incoming server requests.
-#server.maxPayloadBytes: 1048576
+# Modified the payload limit to accommodate file payloads of max 25 MB for uploading custom geojsons in region map plugin.
+server.maxPayloadBytes: 31457280 # 30 MB
```

Comment:
As discussed offline, let's not update this here, instead, mention this in documentation.

Reply:
Changed this to route specific configuration in `server/routes/geospatial.ts`.

```
options: {
        body: {
          accepts: 'application/json',
          maxBytes: FILE_PAYLOAD_SIZE, // 25MB payload limit
        },
      }
```

Thanks @vamshin @VijayanB